client/lib/analytics/ad-tracking/record-order.js

```diff
@@ -432,6 +437,61 @@ function recordOrderInGoogleAds( cart, orderId ) {
 	}
 }
 
+function recordOrderInGAEnhancedEcommerce( cart, orderId, wpcomJetpackCartInfo ) {
```

Comment:
Does it make sense to move this other GA event tracker into this method also?

https://github.com/Automattic/wp-calypso/blob/9e68e6e3ca64f6a2dd16c42911b2aaada7a181fd/client/lib/analytics/index.js#L169-L174

We need to keep that event and also add the eCommerce tracking, but nice if we can consolidate these somehow.

Reply:
That might eventually make sense. But I'd rather save that for a later iteration.